bc_obps/registration/api/user_operator.py

```diff
@@ -48,7 +48,8 @@
     response={200: PendingUserOperatorOut, codes_4xx: Message},
     url_name="get_user_operator_from_user",
 )
-@authorize(["industry_user"], UserOperator.get_all_industry_user_operator_roles())
+# brianna I think it's fine for them to see this unapproved
```

Comment:
Thoughts?

Reply:
I think it's fine too!

Reply:
Agreed. This is their own data & contains nothing they couldn't see while creating a request